### PR TITLE
[native] Correct test data path for internal test runs

### PR DESCRIPTION
**Pull request by mshang816**
Correct test data path for internal test runs as the code was moved to a different location.

Test plan - (Please fill in how you tested your changes)
Tested internally. Here we just have to make sure existing CI runs are green.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by amitkdutta — APPROVED** [review]
Thanks for fixing it.

File: presto-native-execution/presto_cpp/presto_protocol/tests/PlanFragmentTest.cpp [hunk 1]
```diff
@@ -25,9 +25,11 @@ using namespace facebook::presto;
 
 namespace {
 std::string getDataPath(const std::string& fileName) {
+  const std::string FBCODE = "fbcode";
```
Comment (mbasmanova) [inline presto-native-execution/presto_cpp/presto_protocol/tests/PlanFragmentTest.cpp:28]: `static const std::string kFbcode = "fbcode";` per coding style

Comment (mshang816) [reply presto-native-execution/presto_cpp/presto_protocol/tests/PlanFragmentTest.cpp]: > `static const std::string kFbcode = "fbcode";` per coding style

fixed.

**Review by mbasmanova — APPROVED** [review]
Thanks. Please, add [native] prefix to commit title and PR title.

**Pull request merged**